{{cookiecutter.extension_name}}/{{cookiecutter.extension_name}}/__init__.py

```diff
@@ -4,16 +4,17 @@
 
 def _jupyter_server_extension_paths():
     return [{
-        'module': '{{ cookiecutter.extension_name|replace("-", "_") }}'
+        "module": "{{ cookiecutter.extension_name|replace("-", "_") }}"
     }]
 
 
-def load_jupyter_server_extension(nb_app):
+def load_jupyter_server_extension(lab_app):
     """Registers the API handler to receive HTTP requests from the frontend extension.
+
     Parameters
     ----------
-    nb_app: notebook.notebookapp.NotebookApp
-        Notebook application instance
+    lab_app: jupyterlab.labapp.LabApp
```

Comment:
In the near future this will be the Jupyter Server app [itself](https://github.com/jupyter/jupyter_server/blob/3a75adabef5003338246fb767760a0549601e654/jupyter_server/extension/application.py#L414), but this is fair to keep for now.

Reply:
Ok I keep it like this for now.